server/app.js

```diff
@@ -21,6 +21,7 @@ const app = express();
 const bodyParser = require('body-parser');
 app.use(bodyParser.json());
 app.use(require('./routes/jwtAuth'))
+//app.use(require('./routes/dashboard'))
```

Comment:
It's usually preferred to remove no longer needed code, rather than to comment it out. If you need it again in the future you can look in the Git version control after all 👍

Reply:
Hi @petemcfarlane, I will try to implement the changes you have suggested tomorrow and If I stuck up with anything I will ask for your help in slack.
1. I didn't touch eslint because there was a thread about it  in north west class-3 which I wasn't sure about it , but definitely do some research about it.
2.About api.js  , as I am planning to have separate file for each routes so I  made  a separate folder.I still kept the original as it came with project template.Feel free to correct me it is incorrect.
Once again thank you very much for your valuable feedback .It is really appreciated.

Reply:
es-lint is sorted as well .I enable it .It was  this suggestion by our classmates , eslint shows linebreak errors almost on each line. it is a bit annoying.  you can change linebreak style by adding that code "linebreak-style": 0,  to the .eslintrc file.
So I managed to follow his instruction and it is pretty much sorted.Thank you.